### PR TITLE
chapter 4

### PR DESCRIPTION
**Pull request by hoangquangchinh**
Screenshot from 2020-07-29 14-28-25
Ready

### CODE REVIEW

**Comment by hoangquangchinh** [comment]
ready

**Comment by hoangquangchinh** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (hieplt-0914) [inline app/helpers/application_helper.rb:3]: I18n em

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+    if page_title.empty?
```
Comment (hieplt-0914) [inline app/helpers/application_helper.rb:4]: Dùng toán tử 3 ngôi

**Comment by hoangquangchinh** [comment]
ready

**Pull request merged**